tests/Unit/Domain/DomainCreators/Test_Brick.cpp

```diff
@@ -167,6 +168,24 @@ SPECTRE_TEST_CASE("Unit.Domain.DomainCreators.Brick", "[Domain][Unit]") {
            {Direction<3>::lower_zeta(), {0, aligned_orientation}},
            {Direction<3>::upper_zeta(), {0, aligned_orientation}}}},
       std::vector<std::unordered_set<Direction<3>>>{{}});
+
+  // Test serialization of the map
+  DomainCreators::register_derived_with_charm();
+
+  const auto base_map =
+      make_coordinate_map_base<Frame::Logical, Frame::Inertial>(
+          AffineMap3D{AffineMap{-1., 1., lower_bound[0], upper_bound[0]},
+                      AffineMap{-1., 1., lower_bound[1], upper_bound[1]},
+                      AffineMap{-1., 1., lower_bound[2], upper_bound[2]}});
+  const auto base_map_deserialized = serialize_and_deserialize(base_map);
```

Comment:
In order to test the serialization, shouldn't we be checking that base_map_deserialized == base_map?